### DISCUSSION TITLE
Import songs from Database

### DISCUSSION THREAD

**Discussion by yurividal** [ChurchApps/ChurchAppsSupport#389] [FreeShow]
Hi. Firt of all, thanks for this amazing app.
My church is currently trying to migrate away from OpenSong - it hasn't been updated in years, and we are starting to have a lot of crashes with it. 
We have a huge database in opensong with all the songs we use at church, some of them are customized.
It doesn't seem like its currently possible to import any songs into freeshow?
Is this a feature on the roadmap?

**Comment by vassbo** [comment]
Great! You can import your songs, File>Import>OpenSong

**Comment by yurividal** [comment]
@vassbo 

Thanks for the response. I tried this, but i think I might be doing something wrong. The songs all import into single slides. 

So, instead of having one slide per verse, I have all verses on the same slide.
Freeshow doesn't seem to understand the [V1][V2][C] syntax...

**Reply by yurividal** [reply]
![image](https://github.com/user-attachments/assets/1554d1de-ac70-47f0-a274-a16d07387692)

**Comment by vassbo** [comment]
It's missing line breaks between the sections. Let me add a check for that.
